solvers/snapml.py

```diff
@@ -13,13 +16,15 @@ class Solver(BaseSolver):
 
     def set_objective(self, X, y, lmbd):
         self.X, self.y, self.lmbd = X, y, lmbd
+        self.use_gpu = cuda_version is not None
```

Comment:
can you make GPU a parameter of the Solver so we can run both in one benchmark? you should just skip the parameter gpu=True when cuda is not available. Please also do this on the Lasso benchmark. 🙏